package.json

```diff
@@ -38,16 +40,7 @@
   "engine": {
     "node": ">=0.8.x"
   },
-  "licenses": [
-    {
-      "type": "GPL",
-      "url": "https://github.com/wikimedia/jquery.i18n/blob/master/GPL-LICENSE"
-    },
-    {
-      "type": "MIT",
-      "url": "https://github.com/wikimedia/jquery.i18n/blob/master/MIT-LICENSE"
-    }
-  ],
+  "license": "(MIT OR GPL-2.0)",
```

Comment:
According to http://spdx-legal.spdx.narkive.com/NGgCHfMa/spdx-2-0-spec-draft-seeking-community-feedback-for-the-license-expression-syntax-section the "OR" is correct.

GPL-2.0 is missing "+".